static/index.html

```diff
@@ -112,9 +112,9 @@
                         </div>
                         <p>Introducing the NIL, a token with a price set at zero.</p>
                         <p>During the Token Distribution Event
-                            (TDE), those who send 0 ether to the contract address will have back 1000 NIL. To
-                            have
-                            more NIL, just send 0 ether again, and so on until reaching the maximum per wallet.</p>
+                            (TDE), those who send 0 ether to the contract address will receive 1000 NIL in exchange. To
```

Comment:
should this file be `.gitignore`d

Reply:
also the bundle.min.js?

Reply:
There was a reason for that. But recently I have changed a few things and probably that is no more necessary. I will check it. Thanks.